### PR TITLE
Hotfix#70 experimental

### PR DESCRIPTION
**Pull request by gnos28**
à revoir ensemble c'est une galère

### CODE REVIEW

**Review by PaulKip44 — APPROVED** [review]
"experimental"

File: frontend/src/components/Satellites/GlobeContainer.jsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+      tooltip.satname = d.satname;
+      tooltip.satalt = d.satalt;
+      tooltip.satcat = d.category_name;
+      tooltip.satLaunchDate = d.satLaunchDate;
+      tooltip.satObsDate = d.satObsDate;
+
+      //   setTooltip(tooltipDatas);
+      // hoverInfo = { ...tooltipDatas };
+      console.warn(tooltip); // juste pour la démo !!
```
Comment (lchastanet) [inline frontend/src/components/Satellites/GlobeContainer.jsx]: Would it block the team ?

File: frontend/src/components/Satellites/HoverInfo.jsx [hunk 2]
```diff
@@ -1,22 +1,42 @@
...
+  // });
+
+  useEffect(() => {
+    // const now = new Date().getTime();
+    // console.log("HoverInfo try to render", now - lastUpdateTime.current);
+    // if (now - lastUpdateTime.current > 500) {
+    // console.log("HoverInfo should render", now - lastUpdateTime.current);
+    setRender(!render);
+    // lastUpdateTime.current = new Date().getTime();
+    // }
```
Comment (lchastanet) [inline frontend/src/components/Satellites/HoverInfo.jsx]: Same here

File: frontend/src/pages/Satellites.jsx [hunk 3]
```diff
@@ -124,8 +86,11 @@ export default function Satellites() {
   };
 
   useEffect(() => {
+    controller = new AbortController();
+    const { signal } = controller;
+
     axios
-      .get(`http://localhost:5000/api/n2yo/catbysatid`)
+      .get(`http://localhost:5000/api/n2yo/catbysatid`, { signal })
```
Comment (lchastanet) [inline frontend/src/pages/Satellites.jsx]: This will never work when deployed

**Review by lchastanet — CHANGES_REQUESTED** [review]

**Comment by gnos28** [comment]
latest commit is ok

**Comment by gnos28** [comment]
j'ai re git pull push

**Review by lchastanet — APPROVED** [review]
Ok

**Pull request merged**